api-reference/50 Common/Object Structures/positionConfig/boundaryOffset/boundaryOffset.md

```diff
@@ -4,25 +4,113 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-Specifies the horizontal and vertical offset from the window's boundaries.
+Specifies the offset of the boundaries from the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) element.
 
 ---
-This option is used to resolve collisions. If the specified [offset](/api-reference/50%20Common/Object%20Structures/positionConfig/offset '/Documentation/ApiReference/Common/Object_Structures/positionConfig/offset/') from the specified [target](/api-reference/50%20Common/Object%20Structures/positionConfig/of.md '/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of') leads to a collision with the window's boundary, the **boundaryOffset** value is used to position the element near the place of collision.
+The offset is specified in pixels. To set this option, use an object with the **x** and **y** fields or a string value that indicates the horizontal and vertical offsets separated by a whitespace character. Positive offsets narrow the boundaries, negative offsets widen the boundaries.
```

Comment:
Can you be more specific than "use an object with the..."?
For example, do you mean:
To set this option, specify the object's x and y fields, or use a string value that defines horizontal and vertical offsets separated by a whitespace character.
If not, please suggest a revised version.

Change the last sentence to:
Positive offsets narrow boundaries; negative offsets widen boundaries.

Reply:
Do you have an update on the above comments?